externs/olx.js

```diff
@@ -2386,6 +2386,26 @@ olx.interaction.DragBoxOptions.prototype.condition;
 
 
 /**
+ * A function that takes a {@link ol.MapBrowserEvent} and two
+ * {@link ol.Pixel}s and returns a `{boolean}`. If the condition is met,
+ * true should be returned.
+ * @typedef {function(ol.MapBrowserEvent, ol.Pixel, ol.Pixel)}
+ * @api
+ */
+olx.interaction.DragBoxOptions.BoxEndConditionType;
+
+
+/**
+ * A function that takes a {@link ol.MapBrowserEvent} and two
+ * {@link ol.Pixel}s to indicate whether a boxend event should be fired.
+ * Default is {@link ol.interaction.DragBox.defaultBoxEndCondition}
```

Comment:
This default won't show up in the docs. Better show the default as code block in this jsdoc comment.